### PR TITLE
NEW RLQ testcase using new ID: 6

### PR DESCRIPTION
**Pull request by harinineelu**
Includes the dafault.config file.

### CODE REVIEW

**Comment by janetliuboston** [comment]
Logic looks good to me. Just a few small comments.

File: src/harness/testcases/WINNF_FT_S_RLQ_testcase.py [hunk 1]
```diff
@@ -461,3 +461,155 @@ def test_WINNF_FT_S_RLQ_5(self):
...
+    # due to the 'REMOVE' keyword.
+    relinquish_c = {'grantId': 'REMOVE'}
+
+    # Create the actual config.
+    devices = [device_a, device_c, device_b]
+    grant_requests = [grant_a, grant_c, grant_b]
+    # The CBSD ID from the 0th device (i.e. device_a) in the registration
+    # request and the Grant ID from the 0th grant request (corresponding to the
+    # 0th CBSD) will be used in this relinquishment request.
+    relinquishment_request1 = [{'cbsdId': 0, 'grantId': 0}]
```
[the remark this answers is not in the document]
Comment (harinineelu) [reply src/harness/testcases/WINNF_FT_S_RLQ_testcase.py]: Done.

File: src/harness/testcases/WINNF_FT_S_RLQ_testcase.py [hunk 2]
```diff
@@ -461,3 +461,155 @@ def test_WINNF_FT_S_RLQ_5(self):
...
+    relinquishment_request2 = [relinquish_a, relinquish_c, relinquish_b]
```
[the remark this answers is not in the document]
Comment (harinineelu) [reply src/harness/testcases/WINNF_FT_S_RLQ_testcase.py]: Done.

File: src/harness/testcases/WINNF_FT_S_RLQ_testcase.py [hunk 3]
```diff
@@ -461,3 +461,155 @@ def test_WINNF_FT_S_RLQ_5(self):
...
+    config = {
+        'registrationRequest': devices,
```
[the remark this answers is not in the document]
Comment (harinineelu) [reply src/harness/testcases/WINNF_FT_S_RLQ_testcase.py]: Done

File: src/harness/testcases/WINNF_FT_S_RLQ_testcase.py [hunk 4]
```diff
@@ -461,3 +461,155 @@ def test_WINNF_FT_S_RLQ_5(self):
...
+        'conditionalRegistrationData': conditionals,
+        'grantRequest': grant_requests,
+        'relinquishmentRequestFirst': relinquishment_request1,
```
[the remark this answers is not in the document]
Comment (harinineelu) [reply src/harness/testcases/WINNF_FT_S_RLQ_testcase.py]: Done.

File: src/harness/testcases/WINNF_FT_S_RLQ_testcase.py [hunk 5]
```diff
@@ -461,3 +461,155 @@ def test_WINNF_FT_S_RLQ_5(self):
...
+        'relinquishmentRequestSecond': relinquishment_request2,
```
[the remark this answers is not in the document]
Comment (harinineelu) [reply src/harness/testcases/WINNF_FT_S_RLQ_testcase.py]: Done.

File: src/harness/testcases/WINNF_FT_S_RLQ_testcase.py [hunk 6]
```diff
@@ -461,3 +461,155 @@ def test_WINNF_FT_S_RLQ_5(self):
...
+    grant_requests = [grant_a, grant_c, grant_b]
+    # The CBSD ID from the 0th device (i.e. device_a) in the registration
+    # request and the Grant ID from the 0th grant request (corresponding to the
+    # 0th CBSD) will be used in this relinquishment request.
+    relinquishment_request1 = [{'cbsdId': 0, 'grantId': 0}]
+    relinquishment_request2 = [relinquish_a, relinquish_c, relinquish_b]
+    config = {
+        'registrationRequest': devices,
+        'conditionalRegistrationData': conditionals,
+        'grantRequest': grant_requests,
```
[the remark this answers is not in the document]
Comment (harinineelu) [reply src/harness/testcases/WINNF_FT_S_RLQ_testcase.py]: Fixed here.

File: src/harness/testcases/WINNF_FT_S_RLQ_testcase.py [hunk 7]
```diff
@@ -461,3 +461,155 @@ def test_WINNF_FT_S_RLQ_5(self):
...
+          'fccMaxEirp': 47
+      })
+
+    # Whitelist user IDs.
+    for device in config['registrationRequest']:
+      self._sas_admin.InjectUserId({'userId': device['userId']})
+
+    # Register devices and get grants
+    cbsd_ids, grant_ids = self.assertRegisteredAndGranted(
+        config['registrationRequest'], config['grantRequest'],
```
[the remark this answers is not in the document]
Comment (harinineelu) [reply src/harness/testcases/WINNF_FT_S_RLQ_testcase.py]: Done.

**Comment by harinineelu** [comment]
Hi Janet, Fixed all the var names per your review comments. Could you pls take a look. Thanks!

**Comment by janetliuboston** [comment]
Looks great. (I had to delete the default.config that was previously generated but this won't impact others who have not run this code yet.) Thank you, Harini!

**Comment by harinineelu** [comment]
Thanks Janet (Updated the default.config too). 

Merged #407

**Pull request merged**